CHANGES.md

```diff
@@ -10,9 +10,9 @@
 ## 10.0.0 (DR10, 2022-12-20)
 
 - Near-final updates for DR10
-  ([PR#165](https://github.com/legacysurvey/legacysurvey/pull/165)):
+  ([PR#165](https://github.com/legacysurvey/legacysurvey/pull/165)) and
+  ([PR#166](https://github.com/legacysurvey/legacysurvey/pull/166)):
```

Comment:
Since we've already tagged 10.0.0, this PR will go into 10.0.1.

Reply:
Good point - fixed.